src/azure-cli/HISTORY.rst

```diff
@@ -73,6 +73,7 @@ Release History
 
 **NetAppFiles**
 
+* Update for R4.5 version of the RP and API version 2019-06-01 including provision of protocol types.
```

Comment:
This should be laser-focused on customer-facing changes:

* `netappfiles create/update`: Added new argument `--protocol-types`.

Reply:
Updated. I hope acceptable.